lib/rubocop/string_util.rb

```diff
@@ -95,7 +95,7 @@ def matching_index_range(origin)
       end
 
       def matching_window
-        @matching_window ||= (longer.size / 2).to_i - 1
+        @matching_window ||= Integer((longer.size / 2)) - 1
```

Comment:
This `to_i` is redundant, because `Integer#/` returns an Integer if the argument is a Integer. So we can just omit the `to_i`.